api/voters/models.py

```diff
@@ -69,9 +69,11 @@ def get_status(self):
         return status
 
     def update_user(self):
-        self.user, created = User.objects.get_or_create(email=self.email)
+        self.user, created = User.objects.update_or_create(
+            email=self.email,
+            defaults=dict(username=self.email),
+        )
```

Comment:
I guess I'm not sure why the old code wouldn't work... but I don't know Django.

Reply:
I _think_ your data got into a weird state where the username was unset, so it was trying to create two users with the same email address, but different usernames.

Reply:
I could buy that, I signed up pretty early on.